dear_petition/petition/api/serializers.py

```diff
@@ -35,6 +35,8 @@ class OffenseRecordSerializer(serializers.ModelSerializer):
     disposition_method = serializers.SerializerMethodField()
 
     def get_offense_date(self, offense_record):
+        if offense_record.offense.ciprs_record.offense_date is None:
+            return None
```

Comment:
What petition did this come up for?

Reply:
I think test_redacted_2 from the ciprs-record test base